synapse/storage/databases/main/events_worker.py

```diff
@@ -2580,3 +2580,59 @@ async def have_finished_sliding_sync_background_jobs(self) -> bool:
                 _BackgroundUpdates.SLIDING_SYNC_MEMBERSHIP_SNAPSHOTS_BG_UPDATE,
             )
         )
+
+    async def get_invite_count_by_user(self, user_id: str) -> int:
+        """
+        Get the number of invites sent by the given user in the past 24 hours
+        """
+        timestamp = self._clock.time_msec() - (24 * 60 * 60 * 1000)  # 24 hours ago
+
+        def _get_invite_count_by_user_txn(
+            txn: LoggingTransaction, user_id: str, timestamp: int
+        ) -> int:
+            sql = """
+                        SELECT COUNT(event_id) FROM events WHERE sender = ? AND type = 'm.room.member' AND received_ts > ? AND state_key != ?
```

Comment:
No need for the extra indentation

```suggestion
                SELECT COUNT(event_id) FROM events WHERE sender = ? AND type = 'm.room.member' AND received_ts > ? AND state_key != ?
```